### PR TITLE
Global logger cleanup

### PR DESCRIPTION
**Pull request by kiwiupover**
Moved ember-metal/logger to packages/ember-logger

@krisselden 

Is there anything I should do to logger file in the PR?

### CODE REVIEW

File: packages/ember-logger/lib/index.js [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { context } from 'ember-environment';
 function K() { return this; }
 
 function consoleMethod(name) {
```
Comment (kiwiupover) [inline packages/ember-logger/lib/index.js]: @krisselden 
Should I pull these methods `consoleMethod` and `assertPolyfill` into a util file?

**Comment by krisselden** [comment]
while we have to export it eventually as Ember.Logger, it is really a console polyfill and should be called ember-console not ember-logger.

File: packages/ember-logger/lib/index.js (hunk 1 above)
Comment (krisselden) [inline packages/ember-logger/lib/index.js]: this is fine as is

**Comment by kiwiupover** [comment]
renamed `ember-logger` to `ember-console

**Comment by krisselden** [comment]
I will try to get emberjs-build updated to get this working today

**Comment by kiwiupover** [comment]
Cheers mate

-- 
Dave Laird
Sent with Airmail

On April 26, 2016 at 1:27:12 PM, Kris Selden (notifications@github.com) wrote:

I will try to get emberjs-build updated to get this working today

File: lib/packages.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ var features = JSON.parse(featuresJson).features;
 var packages = {
   'container':                  {trees: null,  requirements: []},
   'ember-environment':          {trees: null,  requirements: [], skipTests: true},
+  'ember-console':               {trees: null,  requirements: [], skipTests: true},
   'ember-metal':                {trees: null,  requirements: ['ember-environment'], vendorRequirements: ['backburner']},
```
Comment (rwjblue) [inline lib/packages.js]: I believe you need to add `ember-logger` to the list of `requirements` here also.

**Comment by kiwiupover** [comment]
@krisselden all will be good to go once https://github.com/emberjs/emberjs-build/pull/152 is deployed and updated here.

**Comment by kiwiupover** [comment]
@krisselden this should be good to go after CI

**Comment by kiwiupover** [comment]
@krisselden this is now ready to merge

**Comment by krisselden** [comment]
thanks

**Comment by kiwiupover** [comment]
@krisselden what should I work on next?

**Pull request merged**